### PR TITLE
workflows: populate arXiv card with arXiv papers

### LINKED ISSUE #2091: holdingpen dashboard: harvested arxiv papers don't get into the arxiv card
The source is now 'hepcrawl' instead of 'arxiv', so it has to search instead for 'acquisiton_source.source' that is arxiv.

### PR DESCRIPTION
**Pull request by jacquerie**
Closes #2091

### CODE REVIEW

**Comment by jacquerie** [comment]
Tested locally that this works. This will require a reindex of the holdingpen records to work in production.

**Comment by david-caro** [comment]
Tried locally several times, and with the new mapping, the literature submissions end up showing also on the arxiv card

File: inspirehep/config.py [hunk 1]
```diff
@@ -1437,7 +1438,7 @@ def _(x):
             },
             "source": {
                 "terms": {
-                    "field": "metadata.acquisition_source.method",
+                    "field": "metadata.acquisition_source.source",
```
Comment (david-caro) [inline inspirehep/config.py]: There's a bit of confusion on what the 'source' and the 'method' is, for forms for example, the method and the source are the same, but for hepcrawl they are not (that makes sense).

Maybe we should change the method of the submissions for something like 'forms' at some point.

Comment (david-caro) [inline inspirehep/config.py]: @michamos what do you think?

Comment (david-caro) [inline inspirehep/config.py]: Currently is an open string (not enum).

Comment (michamos) [inline inspirehep/config.py]: but isn't `forms` a synonym for `submitter` or `user` or whatever it is we are using?

Comment (jmartinm) [inline inspirehep/config.py]: At the moment, for all the forms we put `submitter` as a method.

Comment (michamos) [inline inspirehep/config.py]: we could rename `submitter` to `forms` or something like that, but not sure if we gain anything.

Comment (david-caro) [inline inspirehep/config.py]: We might have at some other point sumbissions by email, or something.

The idea is to separate the source (user) from the method (form), the same way we separate them for harvests (source:arxiv,pos,wsp..., method:hepcrawl).

Or we can altogether remove one of the (method probably) and just use source.

Comment (kaplun) [inline inspirehep/config.py]: I think this should be moved to a broader discussion on `inspire-schema`, but I prefer not to block this PR.

**Pull request closed without merging**